### ISSUE TITLE
Dark matter, Red matter hammer bug.

### ISSUE THREAD

**Issue by Kekerisz** [xJon/Tekkit-2#193]
**Basic details**
- **Are you using a legitimate Minecraft account?**: yes
- **Are you using Java 8, 64-bit?**: yes
- **Did you add any additional mods**:   no
- **Crash/latest log link (if applicable)**: don't need

--> The log doesn't say anything, when i press the "switch mode" key.

**Help needed**

I can't switch to the mega impact mode on any hammer. I can switch mode on the pickaxe, i can charge it up, as the pickaxe, but i just can't use the mega impanct mode on the hammers.

**Comment by xJon** [comment]
Seems this feature was never implemented into ProjectE.

**Comment by Kekerisz** [comment]
Yeah i did not find anybody on the internet have this issue. I tried to reinstall the modpack, but same result.
